doc/design/subscription-config.md

```diff
@@ -0,0 +1,50 @@
+# Subscription Config
+
+## Configuring Operators deployed by OLM
+
+It is possible to configure how OLM deploys an operator via the `config` field in the [subscription](../../pkg/api/apis/operators/subscription_types.go) object.
+
+Currently, the `SubscriptionConfig` object allows users to configure the following features on Deployments created by OLM:
+
+### Selector
+The `selector` field defines a list of `Label` selectors for pods. Existing ReplicaSets whose pods are selected by this will be the ones affected by this deployment.
+> Note: It must match the pod template's labels.
+### NodeSelector
```

Comment:
Should we document these before they have been implemented?

Reply:
Good point - I had only added them because they are defined in the [subscription_types.go](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/api/apis/operators/subscription_types.go) file.

Reply:
I will update this doc to only include:

- EnvFrom
- Env
- Volumes
- VolumeMounts

It does not appear that any of the other fields are supported yet.